imports/api/transactions/transaction.js

```diff
@@ -814,6 +815,25 @@ const _genesisTransaction = (userId) => {
   _transact(Meteor.settings.public.Collective._id, userId, rules.VOTES_INITIAL_QUANTITY);
 };
 
+/**
+* @summary reads balance from publicAddress and loads it into user wallet
+* @param {string} userId - id of user being generated within collective
+*/
+const _loadExternalCryptoBalance = (userId) => {
```

Comment:
One decision being made with this PR would be modularizing the initial balance loading into this new method (and hence different than _genesisTransaction()). Because the way Metamask users are being created within the app is different than regular users, I'm thinking it makes sense to have this separate method for balance loading. This is also looking into the future, thinking it would add more flexibility when the time comes where this sort of feature will be needed for all users.

Also with this PR the wallet.available variable is being reset every time a user logs in, leaving it as such for testing purposes for now. This is where the daemon would come in and keep numbers in sync.